test/Microsoft.OpenApi.CSharpAnnotations.DocumentGeneration.Tests/ExtensionsTests/StringExtensionTest.cs

```diff
@@ -45,10 +45,33 @@ public static IEnumerable<object[]> GetTestCasesForStringExtensionShouldSucceed(
             };
         }
 
+        public static IEnumerable<object[]> GetTestCasesForRemoveDuplicateStringShouldSucceed()
+        {
+            yield return new object[]
+            {
+                "String with special character $",
+                "$skip$skip",
+                "$skip"
+            };
+
+            yield return new object[]
```

Comment:
Add test cases with multiple special chars

Reply:
sure

---
In reply to: [253638329](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations/pull/198#discussion_r253638329) [](ancestors = 253638329)